applications/nrf5340_audio/src/modules/power_meas.c

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/sys/util.h>
+
+LOG_MODULE_REGISTER(power_meas, CONFIG_LOG_POWER_MODULE_LEVEL);
+
+/** Number of power rails */
+#define POWER_RAIL_NUM 4U
+
+/** Power rail configuration */
+struct power_rail_config {
+	/** Name */
```

Comment:
Nitpick: Comments should be used where they add readability, and in this struct the comments almost repeat the variable names. This will also increase the maintenance workload.

Reply:
removed